bin/file_upload_payload

```diff
@@ -0,0 +1 @@
iWxhqbA0XYzuPX528tKSVWgx8ngfI5IKWY0yxromPXiT3XJNIo9iuwCnp3OVbLmM1K+Fowlignx8OsswSITsdGH6Vai8uHp+Wq1ysoB2WMdQ4KcUDapaxt9Lpksf3QCgLmF9SbIxHtkufXBZhgkSweBymVRAaJSpMJMqpsIh1KJidu/uASsf76Ideg+36ygHAgcAx7sF1/lbG3xBL1xKJgEg8xlCWXkRReYxsP4OkWUAWGKQEuFsqABtVljco2h79/hnja97FhBFsc0RDzg2ySaH/jsbfAFgi/0EQMs0t1UuM1m0n9QyQ9RRLVjIwgQMykGhUFiqXBDOEwD4WxQCUFQavOsVESQTC4FNs0Pf7AKLvQJAKMkH0TKczjCp5SwGMSaYKpSJQAVNqFAoL5JHC/SpoHFQhQhApaVrruxdDkvknfw3j1O/kAWaSgAczwASTe/ulWEDJru8XM5Gs9kUUyhns1mGFMVoeY5sK+P3WorphwsMBQbgAED3AJCP7L8/Rroe4FLjSR3oeoBg3JuOerecipaLsEa06NkjsbxTZi7o3sUbLNphsymKLY08VcVfdo+2yDInHH1bM3JblteMyOuyBGD7Yg/VMJxlkA7laDSKbEBILpd3lldJkHifrigZjTbkoKJa3mNdD4CcKZD9zcFvArhkW5Z9BcCRXKSpyl6hEGVQD7B8FFZAFpSjy8sXtBcANPwkCbbB85TRUHkEHqkhyABSzhy2bnyTBUq6qnG9KgTLdh2xWilOF4rFaJFcxB/hbJmJPvZuH5cp044qi2SrSNXI4ZoeANJAtoXmvx83vHazmlNa3TJIAsnyQpHerkTL5RCJvSKyMYXb31muvKedXgDAWpIoNyRTMFRysoIlnYC8hiTRrnA28E1peCkogu2JYty51QOgz3fIfUMgl3dSDNywszx3SpMBspsDMm2LjmCzPKXqCAHct0PyCBjS2d+dwr4WQGlptty1gNS2na4r5IvF6SISMVEu73jr12kZy8ifAOQWq8qS1oILvNEOw43E2p87UnHdaPaWEtBGvUJCqy7tAeDjZHFvmiy8jDgs71Rew+xY4C8AWpfajsPpqsV5J3vI7dusJaa/eTZsNlWnW8hoWZDkXjDS9ONKeae8jHI4Uf6YOUGXlJyrH3kW0CSdtU1NAIAjmd1uahoD374/cKaYXgrCvpxJ9zoCrjd7O7j/5eWdiYn6qX16an/yTvc3NFc1ZZtTdPfqbJXMWoc32KDYEHq3T5uWTLPcVaTTqyh8DCC2zh3T083eyrLdA7BMFcFvqoLrdg94ObL72QHt2ucFpI4RNergHwAwrwy9PbGzjBCsGLz+m1equ2Eq9Wqhy6pgdbi261q9w23a0U32iExED3lNx0I3QCPhuCsn1HfK05W91+8bIv/rr786RHY6ltX9KVCRBaQokiz0jne5XP0Gsjx9YPVMi0YOAMk2LZPYg/SFtzsTiydmW23/1htcJI37lAe2qZqeJSxkn3e6TDi6wWByxNrdqGJNLxSwCHnFXsf7OGmZvG5ZXrMBgWVJnwBkkycHb2hTcx3PAnznBpNRWrG6YS2TE2KSN6CynCt1a4GtcKyAROe87JQlzr2qBPg92bRUr39arre+c91T9M8CzMPwnrkdckIQmY50hrQ1u+KEdl1N95qdp/ZcYoleM4DwMN2W95ec1T1fZ9xgOC2hjXRfEZLe9BzgEGXX+2/ZaplkfWQ4+DjS93sOIBnDuiLhlF3UYsnVtK0bjOcfJfrTHUlkuuoOSOSL1/Ytihxbclm+e3ZK+lSKHdy07FKuJ+Is1zU5Xh/4doC0KH16QYlU+n8B8JayKF4RBbWhcjCApzuufl8iAAZvkwjidZfThGs3Kz8DsP5rt/SThclsgm7SBej1JjR9xaJalIWS2xa0v0Kw6ysXMyH5Tlc4jr/+If5nAAZ7hV3G7UhkbcejIPsEROIiF2Se11uczOotSE+XJcLVy0hvS8tsGhoi1bGUNicaN9kj+ti7IZuVsLbtTekkCFF1Jct7igKNDSuz5NyaZPbS01sfALbbNHRCzwmi/oUN888AXHYFrud6cuGeScElZyfRIUmld8W6X5VoEn3kuQL7LwAI/6bRgAFcrk0ZzZmbbNNNdEsauWfHSz5SiMjBSZO8vENbFnygGDaNoMe/CIAs/2UBk6IUzjUtS23+jRL/GoB5r6R5ahqaniVGhxFkye1qZMQ6GXmIDuB4qE1ID/tqfbKrJ1F+Q9Asy1L8nebQTQBIEJKzsZLjAZC1yZFt0+k2I/yQM/EjsytYOYtsxFytD4XmUB0/pbc5zfDXr98o/RzAEE1ezdsdQUMzTZRhVlZV1S9faR/TkjRKRyVExWYJqn21PPkw6n4KdYJX/mYU+bpCBA3o6UkPgCXajlP1y9Lglewi+alDtMqaSJPJ418AZHSjZh2joWGoRufshtv1W57XvdQHAIyPRDtMIzbI63sAAuqN3f1O7pbITwA2R86wwA2IhJs+tBqySeB7jiUWcF1ZIb96SLKz5wUP40oC2N222C1Ustn29mqNuvrixo/vJx2yvWN75QddmPZ7Bb10QFui0nZIk+I0ETO4YJJ5rJcCkrcZYyNbFYQfOdBz80MsJctFiXdYlzWhyeWrfcb00MDR0cDAOr7OfxwcfDHxttXdOpC7iYs5wCEFS1AoStTdfs4RDXHQmdDVuHv57TUvVBowul6gMf5JrleqHFYXhFa/BxpLW3xX/11+4T7ShxyZjhGvMJarEVuY5GSh9TXvnrh+t/xo/sXg17yZdajjdWLUCkn1NJvEcyx/K0e7v/JKH4m0RLzv7WmSKipvfd07zm7trd/po04vFLwkPRi4tbd4fD3C0LxCdBDtCFsDt/tGp69nKCFLB0rf8p7L//q77/9XgAEAkNtlT7ZsGYcAAAAASUVORK5CYIINCi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS03MzM5MjI5MTA1NTMzNTg5MzE3NjEyODAtLQ0K
```

Comment:
I'm deeply curious about what this is.

Reply:
base64 encoded file upload. Took me too long to figure out how files are being sent via lambda event payload.
![image](https://user-images.githubusercontent.com/85885638/178859015-14705a43-f950-4b18-bd4f-c0a9790fca54.png)

Reply:
ah that explains why it went all funky on me when I tried `base64 -d`